site/src/modules/dashboard/Navbar/NavbarView.tsx

```diff
@@ -167,16 +175,79 @@ const NavItems: FC<NavItemsProps> = ({ className }) => {
 			>
 				Templates
 			</NavLink>
-			{metadata["tasks-tab-visible"].value && (
-				<NavLink
-					className={({ isActive }) => {
-						return cn(linkStyles.default, isActive ? linkStyles.active : "");
-					}}
-					to="/tasks"
-				>
-					Tasks
-				</NavLink>
-			)}
+			<TasksNavItem user={user} />
 		</nav>
 	);
 };
+
+type TasksNavItemProps = {
+	user: TypesGen.User;
+};
+
+const TasksNavItem: FC<TasksNavItemProps> = ({ user }) => {
+	const { metadata } = useEmbeddedMetadata();
+	const canSeeTasks =
+		!!metadata["tasks-tab-visible"].value ||
+		process.env.NODE_ENV === "development";
+	const filter = {
+		user: {
+			label: user.username,
+			value: user.username,
+			avatarUrl: user.avatar_url,
+		},
+	};
+	const { data: idleTasks } = useQuery({
+		queryKey: ["tasks", filter],
+		queryFn: () => data.fetchTasks(filter),
+		refetchInterval: 10_000,
```

Comment:
Can we use a longer fetch interval and conditionally refetch if an action to create a workspace / task was performed in the last fetch interval?

Reply:
> Can we use a longer fetch interval 

Yeap! Wdyt about one minute?

> Conditionally refetch if an action to create a workspace / task was performed in the last fetch interval?

Hm... I think so, but it looks complex. IMO, we could increase the refetch time and see if it is necessary to optimize it to avoid adding unnecessary complexity. Thoughts?